src/test_application/activescenes/scenarios_universetest.cpp

```diff
@@ -22,6 +22,7 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
+#if 0
```

Comment:
just delete files that you're deleting. having a bunch of stuff commented out, or disabled with #if 0, only adds to the confusion of other contributors.